### PR TITLE
default to sql

### PR DESCRIPTION
**Pull request by jmtroth0**
I noticed new domains appearing that use couch and was a bit confused, so this should stop that. The first commit is the interesting one here. 

It looks like it should be safe to me, since the apps that people are copying over shouldn't be using too many "advanced features". Let me know if that's a bad assumption.

@snopoke @emord @gcapalbo

### CODE REVIEW

File: corehq/apps/domain/models.py [hunk 1]
```diff
@@ -713,6 +713,7 @@ def save_copy(self, new_domain_name=None, new_hr_name=None, user=None,
             new_domain.internal = InternalProperties()
             new_domain.creating_user = user.username if user else None
             new_domain.date_created = datetime.utcnow()
+            new_domain.use_sql_backend = True
```
Comment (emord) [inline corehq/apps/domain/models.py]: How was this being set before?

Comment (jmtroth0) [reply corehq/apps/domain/models.py]: It wasn't, which makes me thing it was just being copied over https://github.com/dimagi/commcare-hq/blob/4b4e8d102cbcd3cc68732f1e2fa41a10ac28df70/corehq/apps/domain/models.py#L692

Comment (emord) [reply corehq/apps/domain/models.py]: hmm. maybe we should run manual migration to set https://github.com/dimagi/commcare-hq/blob/4b4e8d102cbcd3cc68732f1e2fa41a10ac28df70/corehq/apps/domain/models.py#L272 and change the default property to `True`

Comment (jmtroth0) [reply corehq/apps/domain/models.py]: >run manual migration

How do you mean?

Comment (emord) [reply corehq/apps/domain/models.py]: Offlined and this is probably unreasonable

**Review by emord — APPROVED** [review]

**Pull request merged**